distribution/docs/src/main/resources/content/_using/view-histogram.adoc

```diff
@@ -0,0 +1,10 @@
+:title: View Histogram
+:type: subUsing
+:status: published
+:parent: Map Context Menu
+:summary: View a search histogram
+:order: 02
+
+== {title}
+
+Select a search, right click on the map view, and select *View Histogram* to view a histogram for the search.
```

Comment:
Do we know what these histograms are supposed to show? I personally never understood what data they're used for or how it would be used. Maybe we can explain what it's for or link it to somewhere in the docs that mentions it's usage

Reply:
I added an image, hopefully that makes things a little clearer?